gui.py

```diff
@@ -0,0 +1,75 @@
+import tkinter as tk 
+
+LARGE_FONT= ("Verdana", 12)
+
+
+class Gui(tk.Tk):
+
+    def __init__(self, *args, **kwargs):
+        
+        tk.Tk.__init__(self, *args, **kwargs)
+        container = tk.Frame(self)
+
+        container.pack(side="top",expand = True)
+
+        container.grid_rowconfigure(0, weight=1)
+        container.grid_columnconfigure(0, weight=1)
+
+        self.frames = {}
+
+        for F in ([StartPage]):
+
+            frame = F(container, self)
+
+            self.frames[F] = frame
+
+            frame.grid(row=0, column=0, sticky="nsew")
+
+        self.show_frame(StartPage)
+
+    def show_frame(self, cont):
+
+        frame = self.frames[cont]
+        frame.tkraise()
+
+
+#FORMATTING FUNCTION FOR THE TKINTER ENTRY BOX 
+
+             
+class StartPage(tk.Frame):
+
+    def __init__(self, parent, controller):
+        def on_entry_click(event):
+            """function that gets called whenever entry is clicked"""
+            if URIentry.get() == 'https://cs.odu.edu/':
```

Comment:
Adding hardcoded values like this is not a good practice. If this were to be used as the sample URI, a constant should be defined at the top of the file or better yet, add this in the config file to reuse everywhere.